bpfman-api/src/config.rs

```diff
@@ -63,6 +63,15 @@ impl XdpMode {
             XdpMode::Hw => XdpFlags::HW_MODE,
         }
     }
+
+    pub fn from_u32(val: u32) -> Self {
```

Comment:
Convert to tryfrom

Reply:
Thanks! Saved me adding that comment here lol